Doc/library/graphlib.rst

```diff
@@ -194,15 +196,74 @@
    .. versionadded:: 3.9
 
 
+Graph Functions
+---------------
+
+Some functions are provided to work with directed acyclic graph structures using
+the same representation accepted by :meth:`TopologicalSorter.__init__`: a
+mapping of nodes to iterables of predecessors.
+
+Nodes can be any :term:`hashable` object. Duplicate edges are ignored.
+
+For example, it can be useful to construct a TopologicalSorter that processes
+a graph in reverse order. This can be done by passing the :func:`reverse` of a
+graph to the constructor of :class:`TopologicalSorter`::
+
+   ts = TopologicalSorter(reverse(graph))
```

Comment:
```suggestion
graph to the constructor of :class:`TopologicalSorter`:

.. code-block:: python

   ts = TopologicalSorter(reverse(graph))
```

Reply:
Why? This is equivalent but 2 lines longer

Reply:
We have a slight preference for the more explicit `code-block` directive in new documentation, though as you say both have the same result.

Reply:
Updated to use this throughout.